routers/web/repo/issue.go

```diff
@@ -803,6 +803,9 @@ func NewIssue(ctx *context.Context) {
 			ctx.Data["Project"] = project
 		}
 
+		if len(ctx.Req.URL.Query().Get("project")) > 0 {
```

Comment:
It seems this is unnecessary? The POST requests will also request the current URL. So that the query strings are still there.

Reply:
No, the form's `action` is filled by `{{.Link}}`, without query string. So this parameter is necessary.